tozti/__main__.py

```diff
@@ -80,6 +80,40 @@ def find_exts():
             continue
 
 
+def load_config_file(path = "config.toml"):
+    """Load tozti's configuration file.
+    TODO: add validation
+
+    Args:
+        path (str): the path where the config file should be
+
+    Returns:
+        a dictionnary representing the config file
+
+    Raises:
+        Exception if the file couldn't be loaded
+    """
+
+    # put this in a function for the moment when validation will be done
+    config = {}
+    required = {"http":["host", "port"], 
+                "mongodb":["host", "port"], 
+                "cookie": ["private_key", "public_key"]}
+    with open(path) as s:
+        config = toml.load(s)
+
+    for major in required:
+        if not major in config:
+            raise Exception("Expected a {} entry".format(major))
```

Comment:
It's not good style to raise `Exception`, either raise `ValueError` or create some `ConfigError` to have meaningful exceptions.

Reply:
@poechsel this is your code, can you fix that ?